Widgets/RecentDonations.html

```diff
@@ -0,0 +1,34 @@
+<html>
+	<head>
+		<script type="text/javascript" src="../bower_components/jquery/dist/jquery.min.js"></script>
+		<script type="text/javascript" src="../bower_components/angular/angular.min.js"></script>
+		<script type="text/javascript" src="../js/Globals.js"></script>
+		<script type="text/javascript" src="../js/RecentDonations.js"></script>
+
+		<link href="http://fonts.googleapis.com/css?family=Cantarell:700" rel="stylesheet" type="text/css">
+		<link href="http://fonts.googleapis.com/css?family=Cantarell:400" rel="stylesheet" type="text/css">
+
+		<link rel="stylesheet" type="text/css" href="widget.css">
+	</head>
+	<body>
+		<div id="widget" ng-app="RecentDonations" ng-controller="RecentDonationsControl">		
+
+			<h1>Recent Donations for {{ displayName }} </h1>
+
+			<div id="error" ng-show="error" ng-repeat="problem in errors">{{ problem }}</div>
+
+			<table ng-hide="error">
+		    <tr>
+				<th>Name</th>
+				<th>Amount</th>
+			</tr>
+			  <tr ng-repeat="x in contributions">
+			    <td class="expand">{{ x.name }}</td>
+			    <td class="rightAlign">{{ x.amount }}</td>
+			  </tr>
+			</table>
+
+			<div id="message">{{ message }}</div>
+		</div>
+	</body>
+</html>
```

Comment:
can you add a newline to make github happy, seeing the
![image](https://cloud.githubusercontent.com/assets/3931713/10120982/c3430eb4-64a4-11e5-87bc-c02c0b44911d.png)
makes me sad...